packages/web-components/src/components/va-process-list/va-process-list-item.tsx

```diff
@@ -45,9 +45,8 @@ export class VaProcessListItem {
     
     return (
       <Host role="listitem" class='usa-process-list__item'>
-        {
-          checkmark || active || pending ? 
-            <span class='sr-only'>{checkmark ? 'Completed:' : active ? 'Current Step:' : pending ? 'Pending:' : null}</span>
+        { checkmark || active || pending ?
```

Comment:
This is a lot of conditionals, I wonder if moving this out of the JSX portion and into a variable or function would improve readability? Something like:

```
function eyebrowText() {
  if (checkmark) {
    return 'Complete';
  } else if (active) {
    return 'Active';
  } else if (pending) {
    return 'Pending';
  } else {
    return null;
  }
}
const eyebrow = eyebrowText();

{eyebrow && <div ...>{eyebrow}</div>
```
I know it'd be a good bit more verbose, but easier to grok?